src/main/java/Duke.java

```diff
@@ -1,10 +1,63 @@
-public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-    }
-}
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Scanner;
+
+import duke.DukeException;
+import duke.Parser;
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+
+
+public class Duke {
+    private Parser parser;
+    private TaskList tasks;
+    private Ui ui;
+
+    /**
+     * Constructor for Duke class.

```

Comment:
Minor issue but I think for JavaDocs there should be an empty line between the description and parameter section